datafusion-cli/Cargo.lock

```diff
@@ -454,17 +454,6 @@ version = "1.1.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1505bd5d3d116872e7271a6d4e16d81d0c8570876c8de68093a09ac269d8aac0"
 
-[[package]]
-name = "atty"
```

Comment:
nice! we finally have removed this old dependency and avoid https://rustsec.org/advisories/RUSTSEC-2021-0145.html